pkgs/shells/bash/default.nix

```diff
@@ -113,6 +113,6 @@ stdenv.mkDerivation rec {
   };
 
   passthru = {
-    shellPath = "/bin/bash";
+    shellPath = "${bash}/bin/bash";
```

Comment:
Yeah I read the part about `passthru` but I don't know the relevancy for `bash.shellPath` ;)